<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDialog</class>
 <widget class="QDialog" name="AddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputPassword"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Группа прав</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboGroupName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>200</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonAddUser">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCloseDialog">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>AddUserDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddUserDialog</sender>
   <signal>finished(int)</signal>
   <receiver>AddUserDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
